invenio_rdm_records/fixtures/communities.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Communities fixture module."""
+
+from invenio_access.permissions import system_identity
+from invenio_communities.proxies import current_communities
+from invenio_pidstore.errors import PIDAlreadyExists
+
+from .fixture import FixtureMixin
+
+
+class CommunitiesFixture(FixtureMixin):
+    """Communities fixture."""
+
+    def create(self, entry):
+        """Load a single community."""
+        access_visibility = entry.get("access").get("visibility")
+        slug = entry.get("slug")
+        metadata_title = entry.get("metadata").get("title")
+
+        community_data = {
+            "access": {
+                "visibility": access_visibility,
+            },
+            "slug": slug,
+            "metadata": {
+                "title": metadata_title,
+            },
+        }
+
+        service = current_communities.service
+        try:
+            service.create(data=community_data, identity=system_identity)
```

Comment:
moderate: could this be done asynchronously to avoid blocking during creation? i.e. in a celery task?

Also, note that there exists a similar functionality in [invenio-communities.cli](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/cli.py#L45), and a [task](https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/fixtures/tasks.py#L29) fot its creation already exists.

While I do agree that maybe we can have the data in this module (tha `.yaml` file), maybe we can use the existing task from communities? all in all just to consolidate the code and avoid duplicates 😇